compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -217,17 +217,31 @@ interface PlatformContext {
         fun onLayoutChange(semanticsOwner: SemanticsOwner, semanticsNodeId: Int)
     }
 
-    companion object {
-        val Empty = object : PlatformContext {
-            override val windowInfo: WindowInfo = WindowInfoImpl().apply {
-                // true is a better default if platform doesn't provide WindowInfo.
-                // otherwise UI will be rendered always in unfocused mode
-                // (hidden textfield cursor, gray titlebar, etc)
-                isWindowFocused = true
-            }
-
-            override val inputModeManager = DefaultInputModeManager()
+    open class Base : PlatformContext {
```

Comment:
Why do we need to declare it inside interface?
Also, missing KDoc

Reply:
We don't *need* to, but I like that it's scoped to `PlatformContext`. In my opinion `PlatformContext.Base` is better than `BasePlatformContext`. It's more discoverable, and it doesn't pollute the global scope.